### DISCUSSION TITLE
Donate/sponsor

### DISCUSSION THREAD

**Discussion by fruktiz32** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#349] [Q&A] [answered]
Hello! I want to thank you creator for this great library I've been using for long time without any issues. For that great job I would like to give some beer money but can not find where I can do that. Please help and thank you again for that work

**Comment by mrcodetastic** [comment, marked as answer]
Thanks for the kind words! Merry Christmas to you!

No need for beer money or anything. Donate what you would have to a worthy cause instead, like https://u24.gov.ua/ or something.

Thanks again for your thoughts though!
